### PR TITLE
Staging changelog for updates

### PR DESCRIPTION
**Pull request by wallentx**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,53 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- `chia show -f` changed to output proper JSON
+- `Rate limiting` log messages are themselves rate limited
+- Notified GUI when wallets are removed
+- Optimized counting of NFTs during removal by leveraging SQLite
+- Offer CLI command help now shows `--fee` units as XCH
+- Optimized offer code by limiting `additions` recomputation
+- `chia_rs` updated to 0.2.4
+- Improve the help text for the `chia peer` and `chia peer -a` commands
+- Remove redundant checks for minting coin and reserve fee
+- `nfg_get_nfts` performance improvments by shifting paging to SQLite
```
Comment (trepca) [inline CHANGELOG.md:33]: `improvements`

Comment (wjblanke) [inline CHANGELOG.md:33]: improvements

**Review by wjblanke — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (wallentx) [reply CHANGELOG.md]: i improvd it

Comment (wallentx) [reply CHANGELOG.md]: i improvd it

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,53 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- `chia show -f` changed to output proper JSON
+- `Rate limiting` log messages are themselves rate limited
+- Notified GUI when wallets are removed
+- Optimized counting of NFTs during removal by leveraging SQLite
+- Offer CLI command help now shows `--fee` units as XCH
+- Optimized offer code by limiting `additions` recomputation
+- `chia_rs` updated to 0.2.4
+- Improve the help text for the `chia peer` and `chia peer -a` commands
+- Remove redundant checks for minting coin and reserve fee
+- `nfg_get_nfts` performance improvements by shifting paging to SQLite
```
Comment (paninaro) [inline CHANGELOG.md:33]:
```suggestion
- `nft_get_nfts` performance improvements by shifting paging to SQLite
```

**Review by wjblanke — APPROVED** [review]
aok

**Review by paninaro — APPROVED** [review]

**Pull request merged**